app/_locales/en/messages.json

```diff
@@ -5733,7 +5736,7 @@
     "message": "Keep the same account address, and you can switch back anytime."
   },
   "smartAccountLabel": {
-    "message": "Smart Account"
+    "message": "Smart account"
```

Comment:
We should be using sentence case here I don't think "smart account" qualifies as a special case cc @coreyjanssen